src/mcts/search.cc

```diff
@@ -235,8 +261,22 @@ void Search::SendUciInfo() REQUIRES(nodes_mutex_) REQUIRES(counters_mutex_) {
     ++multipv;
     uci_infos.emplace_back(common_info);
     auto& uci_info = uci_infos.back();
-    const auto wl = edge.GetWL(default_wl);
-    const auto floatD = edge.GetD(default_d);
+    auto wl = edge.GetWL(default_wl);
+    auto floatD = edge.GetD(default_d);
```

Comment:
I fully agree with both of your points. It's probably still slightly confusing because the different uses of `wl` and `d` in the following code (choice between internal and external) require the distinction?

Reply:
I simplified the "internal d" away a few commits ago, and renamed the wdl output variables to `wdl_w` etc. I consider this resolved now.